ipatests/prci_definitions/nightly_previous.yaml

```diff
@@ -1774,6 +1774,18 @@ jobs:
         timeout: 5000
         topology: *master_2repl_1client
 
+  fedora-previous/test_external_sso:
```

Comment:
Could you use the same label `fedora-previous/test_sso` as in the other files?

Reply:
oops.  that was a search-replace typo.  I'll fix that.

Reply:
Fixed and I dropped the temp_commit.